features/admin.ai.v1/components/loading-screen.tsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Box from "@oxygen-ui/react/Box";
+import CircularProgress from "@oxygen-ui/react/CircularProgress";
+import LinearProgress from "@oxygen-ui/react/LinearProgress";
+import Typography from "@oxygen-ui/react/Typography";
+import useAIBrandingPreference from "features/admin.ai.v1/hooks/use-ai-branding-preference";
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { ReactComponent as LoadingPlaceholder }
+    from "../../../modules/theme/src/themes/wso2is/assets/images/illustrations/ai-loading-screen-placeholder.svg";
+import useGetAIBrandingGenerationStatus from "../api/use-get-branding-generation-status";
+import "./loading-screen.scss";
+
+export const LoadingScreen = (): JSX.Element => {
```

Comment:
Return type should be react element.(`ReactElement`).
Seems like the  component type(`FunctionComponent<..>`) is also missing. Check other components.

Reply:
this is addressed with https://github.com/wso2/identity-apps/pull/5961/commits/f29e43a5208f33565cac518ff47efcdf05dc9b5c. can you please check if the usage is correct?